### ISSUE TITLE
Add example gene set for each species

### ISSUE THREAD

**Issue by ChristopherMancuso** [krishnanlab/geneplexus-app-v2#21]
For the step that validates the genes, need to have a gene set that is specific to the species chosen.

- (Chris) Make species specific example gene sets
- (Vince) Add that function to UI

**Comment by vincerubinetti** [comment]
If you want I can just pick random ones from mygeneset.info that are around 10 genes. That's what I did for the current example jeans.

In my next PR I'll probably do that, at least temporarily until you pick more significant/interesting examples if you want.

**Comment by ChristopherMancuso** [comment]
that would be great to just grab those from mygeneset.info. I probably will find sets that have really interesting biology at some point, but that will be done after more fixing of the python package happens.

**Comment by ChristopherMancuso** [comment]
Arjun provide a possible example in the google doc
[bbs genes](https://drive.google.com/file/d/1GDDQTGO0aMqbRd0B60vyePFkQ3ygsVhz/view)

**Comment by vincerubinetti** [comment]
I just pushed those bbs genes to #38 as new human example genes. (I believe they're human, from googling)

**Comment by ChristopherMancuso** [comment]
sounds good. For the example gene sets I am going to try to find a single biological process that is well annotated in each species so when explaining the results I don't have to learn vastly different biology for each species. On your end I don't think this changes much as I can just point you to those gene sets once I get them, but since we need a walkthrough/tutorial for the NAR submission I think the example gene sets need to be pretty similar to help me out on that.

**Comment by vincerubinetti** [comment]
Marking this as an enhancement since we at least have some serviceable example gene sets from each species. Though they might not provide the most interesting results, the genes themselves are at least inherently related, having been pulled from mygeneset.info from the same set (per-species).

**Comment by ChristopherMancuso** [comment]
I agree, I think this is good for now and the examples can be optimized down the road.
